registries/_format/decimal.md

```diff
@@ -0,0 +1,31 @@
+---
+owner: baywet
+issue: 889
+description: A fixed point decimal number
+base_type: string
```

Comment:
Is the base type string or number?  Suddenly the rest of the page is talking about numbers again.

I am once again confused as to what this registration is attempting to do, and why.  @MikeRalphson did you assessment of `format: decimal` usage include seeing how many people use which underlying type?  If so, does it make sense to specify both here?  If not, @baywet what is driving the dual underlying type?

Reply:
Yep, this is something I forgot to mention in my previous update. Thanks for calling this out. The Azure APIs making use of decimal use it with number. So my suggestion here is to put string|number if that's ok with everyone